src/Resources/contao/dca/tl_page.php

```diff
@@ -1433,11 +1433,9 @@ public function copyPageWithSubpages($row, $href, $label, $title, $icon, $attrib
 			return '';
 		}
 
-		$objSubpages = $this->Database->prepare("SELECT * FROM tl_page WHERE pid=?")
-									  ->limit(1)
-									  ->execute($row['id']);
+		$objSubpages = \PageModel::findByPid($row['id'], array('limit'=>1));
```

Comment:
You could actually use `countBy` here, because we only need to know if there are subpages.

Reply:
But then it's not read from the model registry and an extra sql query is executed (see [Model.php](https://github.com/contao/core-bundle/blob/master/src/Resources/contao/library/Contao/Model.php#L1158)).

Reply:
Well it does not read from the registry this way either... `findByPid` is not cacheable, and nothing is cacheable if you use options.

Reply:
You're totally right. It's only read from registry if a model is requested by it's id.

But one question is left: Is there a (measurable) difference in performance, when setting a limit of one or query all child records and count them? 

I don't think so. The amount of data from the sql server is negligible unless someone have a thousand sub pages. So I'm fine with using `countBy`. Looks more logic.

Reply:
Changed to the `countByPid` method in b89e33070b3e137bdbfcc1929050b2bd27f5a3b5